pkg/broadcaster/unbuffered.go

```diff
@@ -1,18 +1,18 @@
-package broadcastwriter
+package broadcaster
 
 import (
 	"io"
 	"sync"
 )
 
-// BroadcastWriter accumulate multiple io.WriteCloser by stream.
-type BroadcastWriter struct {
+// Unbuffered accumulate multiple io.WriteCloser by stream.
+type Unbuffered struct {
 	mu      sync.Mutex
 	writers []io.WriteCloser
 }
 
 // AddWriter adds new io.WriteCloser.
-func (w *BroadcastWriter) AddWriter(writer io.WriteCloser) {
+func (w *Unbuffered) AddWriter(writer io.WriteCloser) {
```

Comment:
@aaronlehmann we could make it an `io.Writer` and try casting it to `io.Closer` wherever we need to call `Close()` how does that sound?